contrib/test_jenkins.sh

```diff
@@ -942,13 +966,17 @@ run_client_server() {
 			-Wl,-rpath=${ucx_inst}/lib
 	fi
 
-	server_ip=$(get_rdma_device_ip_addr)
-	if [ "$server_ip" == "" ]
+	server_ip_ib=$(get_rdma_device_ip_addr)
+	server_ip_tcp=$(get_tcp_ip_addr)
+	if [ "$server_ip_ib" == "" ] && [ "$server_ip_tcp" == ""  ]
 	then
 		return
 	fi
 
-	run_client_server_app "./${test_name}" "" "-a ${server_ip}" 1 0
+	for server_ip in $server_ip_ib $server_ip_tcp
+	do
+		run_client_server_app "./${test_name}" "" "-a ${server_ip}" 1 0
+	done
```

Comment:
can we simplify this?

Reply:
what do you mean? i wanted it to test both IP's so just added a loop

Reply:
somehow do it at once without for loop and another if